sdks/java/io/google-cloud-platform/src/test/java/org/apache/beam/sdk/io/gcp/bigtable/changestreams/it/BigtableChangeStreamIT.java

```diff
@@ -0,0 +1,361 @@
+import org.apache.beam.sdk.io.common.IOITHelper;
+import org.apache.beam.sdk.io.gcp.bigtable.BigtableTestOptions;
+import org.apache.beam.sdk.io.gcp.bigtable.BigtableTestUtils;
+import org.apache.beam.sdk.testing.PAssert;
+import org.apache.beam.sdk.testing.TestPipeline;
+import org.apache.beam.sdk.transforms.DoFn;
+import org.apache.beam.sdk.transforms.ParDo;
+import org.apache.beam.sdk.values.KV;
+import org.apache.beam.sdk.values.PCollection;
+import org.apache.beam.vendor.guava.v32_1_2_jre.com.google.common.collect.ImmutableList;
+import org.joda.time.Duration;
+import org.joda.time.Instant;
+import org.junit.AfterClass;
+import org.junit.Before;
+import org.junit.BeforeClass;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.JUnit4;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/** End-to-end tests of Bigtable Change Stream. */
+@SuppressWarnings("FutureReturnValueIgnored")
+@RunWith(JUnit4.class)
+public class BigtableChangeStreamIT {
+  private static final Logger LOG = LoggerFactory.getLogger(BigtableChangeStreamIT.class);
+  private static final String COLUMN_FAMILY1 = "CF";
+  private static final String COLUMN_FAMILY2 = "CF2";
+  private static final String COLUMN_QUALIFIER = "CQ";
+  private static String projectId;
+  private static String instanceId;
+  private static String tableId;
+  private static String appProfileId;
+  private static String metadataTableId;
+  private static BigtableTableAdminClient adminClient;
+  private static BigtableDataClient dataClient;
+  private static BigtableClientIntegrationTestOverride bigtableClientOverride;
+  private static Batcher<RowMutationEntry, Void> mutationBatcher;
+  private static BigtableTestOptions options;
+  private transient TestPipeline pipeline;
+
+  @BeforeClass
+  public static void beforeClass() throws IOException {
+    options = IOITHelper.readIOTestPipelineOptions(BigtableTestOptions.class);
+    LOG.info("Pipeline options: {}", options);
+    projectId = options.as(GcpOptions.class).getProject();
+    instanceId = options.getInstanceId();
+
+    long randomId = Instant.now().getMillis();
+    tableId = "beam-change-stream-test-" + randomId;
+    metadataTableId = "beam-change-stream-test-md-" + randomId;
+    appProfileId = "default";
+
+    bigtableClientOverride = new BigtableClientIntegrationTestOverride();
+    LOG.info(bigtableClientOverride.toString());
+
+    BigtableDataSettings.Builder dataSettingsBuilder = BigtableDataSettings.newBuilder();
+    BigtableTableAdminSettings.Builder tableAdminSettingsBuilder =
+        BigtableTableAdminSettings.newBuilder();
+    dataSettingsBuilder.setProjectId(projectId);
+    tableAdminSettingsBuilder.setProjectId(projectId);
+    dataSettingsBuilder.setInstanceId(instanceId);
+    tableAdminSettingsBuilder.setInstanceId(instanceId);
+    dataSettingsBuilder.setAppProfileId(appProfileId);
+    // TODO: Remove this later. But for now, disable direct path.
```

Comment:
Do we still need this? We should get this from the default config now I think, but I don't remember 100% why it was necessary here.

Reply:
We still disable direct path in `BigtableChangeStreamAccessor`. So it would make sense to keep it disabled when we write/read from bigtable.